### PR TITLE
Update node version and all dependencies

### PR DESCRIPTION
**Pull request by reinaldocoelho**
During check of plugin I found a lot of dependencies in older versions, with security risks.

I update an run the project with updates on all libs and Webpack changes.

Running the project works well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,21 +32,21 @@
   "dependencies": {},
   "devDependencies": {
     "quill": "^2.0.0-dev.3",
```
Comment (kagol) [inline package.json:34]: Now Quill 2.0.0 has released, Can you upgrade to the version of Quill 2.0.0? Thank you!

Comment (reinaldocoelho) [reply package.json]: Updated!!

**Comment by Sashkko** [comment]
@soccerloway are there any plans to merge this pull request?

**Comment by soccerloway** [comment]
Thanks for your updates!  Merged.

**Comment by Sashkko** [comment]
@soccerloway thanks for merge! Are there any plans to release a new version for npm?

**Comment by NasirTaha** [comment]
@soccerloway When the npm new version will be release?

**Pull request merged**